charts/woodpecker/charts/agent/templates/statefulset.yaml

```diff
@@ -88,7 +88,7 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      {{- if or (.Values.persistence.enabled) (.Values.extraVolumeMounts) }}
+      {{- if or (and (.Values.persistence.enabled) (.Values.persistence.existingClaim)) (.Values.extraVolumeMounts) }}
```

Comment:
Isn't this a duplicate of https://github.com/woodpecker-ci/helm/blob/bb7452efdac3aea4f0bcf8dcf7b12708b6cc49ff/charts/woodpecker/charts/agent/templates/statefulset.yaml#L93-L94 now?

Reply:
Not quite, l91 is to include/exclude `volumes:` completely and includes `extraVolumeMounts` as an additional condition) while l93 only excludes the pvc mount.

I have now updated the logic, please check again.